public/pug/landing.pug

```diff
@@ -0,0 +1,45 @@
+extends layout
+
+block styles
+    link(rel="stylesheet" type="text/css" href="../css/landing.css")
+
+block main
+    section(class="container")
+        h1  Be more productive
+        p.
+            "What to do" is an easy to use to-do application, designed for
+            individual or colaborativ use.
+        .section-features
+            .features.lists
+                p.
+                    Todos can be grouped into lists in order to sort them.
+                    You want to have seperate lists for work, home and shopping
+                    or a project is around the corner? Sort your task to be able 
+                    to focus on what matters for you.
+                a( href="/welcome" class="btn" )
```

Comment:
Where are these supposed to go?

A sub side describing that part of the functionality?